subxt/src/blocks/block_types.rs

```diff
@@ -92,13 +91,23 @@ where
     pub async fn runtime_api(&self) -> Result<RuntimeApi<T, C>, Error> {
         Ok(RuntimeApi::new(self.client.clone(), self.hash()))
     }
+
+    /// Fetch the block's body from the chain.
+    async fn block_details(&self) -> Result<ChainBlockResponse<T>, Error> {
+        let block_hash = self.header.hash();
+        match self.client.rpc().block(Some(block_hash)).await? {
+            Some(block) => Ok(block),
+            None => Err(BlockError::not_found(block_hash).into()),
+        }
+    }
```

Comment:
This is only used in one place again now so maybe best to put back into `fn body()`?